Dockerfile

```diff
@@ -4,6 +4,8 @@ WORKDIR /tikv
 
 # Install Rust
 COPY rust-toolchain ./
+RUN rustup self update
```

Comment:
em, does it invalidate cache every time?

Reply:
Do you mean tikv build cache? If so, it does not. Because the rustc is locked by `rust-toolchain`.

Reply:
I mean the docker build cache.

Reply:
I don't think so, it works like `apt-get update`. In this case, just the command string itself is used to find a match.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache